### DISCUSSION TITLE
@lit-labs/ssr - Reference to parent or children?

### DISCUSSION THREAD

**Discussion by kyubisation** [lit/lit#4407] [Q&A]
Is it possible with `@lit-labs/ssr` to get a reference to the parent element or children elements?
Judging by the documentation and the code, it does not seem possible, but I wanted to check with the community.

**Comment by augustjk** [comment]
No that is currently not possible as we don't emulate the full DOM in the SSR environment. Any code needing reference to other elements would need to run in client only life cycles.
